release notes/v0.49.0.md

```diff
@@ -0,0 +1,202 @@
+
+For a long time, you could get an Element from a Selection using the [`.get(index)`](https://grafana.com/docs/k6/latest/javascript-api/k6-html/selection/selection-get/), but you couldn't get back to a Selection from an Element.
+
+This is not a common case, but one that requires rewrite of the code quite a lot.
+
+```javascript
+let li = http.get("https://test.k6.io").html().find("li");
+li.each(function(_, element) {
+    // here element is an element not a selection
+    // but what if for each li we want to select something more?
+    // in jquery that will be:
+   let container = $(element).closest('ul.header-icons');
+    // but what should `$` do?
+    // in a browser there is only 1 html document that you have access to
+    // in k6 though you can be working with multiple ones, so `$` can't know which one it should
+    // work against
+});
+```
+
+In order to support the above you can just use the selection, without going to element as in:
+
+```javascript
+let li = http.get("https://test.k6.io").html().find("li");
+for (; li.size() > 0; li = li.next()) {
+    let ul = li.closest('ul.header-icons'); // li here is still a selection and we iterate over it.
+}
+```
+
+This is not always possible though, and arguably isn't what most users will naturally do.
+
+Because of this, we have now added a new [`.selection()`](https://grafana.com/docs/k6/latest/javascript-api/k6-html/element/element-selection/) which returns a selection for it's element.
+
+
+```javascript
+   let li = http.get("https://test.k6.io").html().find("li");
+   li.each(function(_, element) {
+      let container = element.selection().closest('ul.header-icons');
+        // .. more code
+   });
+```
+
+Thanks to @Azhovan! :bow: :tada:
+
+### Collect usage data on imported internal modules and outputs [#3525](https://github.com/grafana/k6/pull/3525)
+
+Added the data collection of the used modules and outputs when the [usage report](https://grafana.com/docs/k6/latest/misc/usage-collection) is enabled. The data collection is only related to the built-in k6 modules and outputs. Private, custom modules and extensions [are never collected](https://github.com/grafana/k6/blob/f35e67902605877ebf2c5e9c8673cd7faf4cdc1e/cmd/report.go#L33-L57). The usage report is enabled by default in k6, but it is possible to opt-out using the [no-usage-report](https://grafana.com/docs/k6/latest/using-k6/k6-options/reference/#no-usage-report) option.
+
+We always want to improve the product, but at the same time we need to pay attention where we allocate our resources. Having data of what are the most used modules and outputs give us a better confidence to make decisions because we are supported by data.
+It will let us know what is the percentage of our users that will benefit from the introduction of a new feature and on the other side, in case of a breaking change, how many of them would be impacted.
```

Comment:
```suggestion
The data can let us know what percentage of our users will benefit from the introduction of a new feature and also, how many of them would be impacted in case of a breaking change.
```